extra-web/thunderbird/autobuild/mozconfig

```diff
@@ -3,7 +3,7 @@ ac_add_options --enable-calendar
 
 ac_add_options --prefix=/usr
 ac_add_options --enable-release
-ac_add_options --enable-linker=gold
+ac_add_options --enable-linker=bfd
```

Comment:
This should not be applied across the board. This is probably only needed with `arm64`?